internal/prep/prep.go

```diff
@@ -123,7 +123,7 @@ func getDeploymentFromDeploymentID(deploymentID string) (string, error) {
 }
 
 func SetupStateroot(log logr.Logger, ops ops.Ops, ostreeClient ostreeclient.IClient,
-	rpmOstreeClient rpmostreeclient.IClient, seedImage, expectedVersion, imageListFile string, ibi bool) error {
+	rpmOstreeClient rpmostreeclient.IClient, seedImage, expectedVersion, tmpPath string, ibi bool) error {
```

Comment:
We should probably rename `tmpPath` to something indicating that this is for IBI. @eranco74 WDYT?

Reply:
Why not keep `imageListFile` ?

Reply:
Because we in addition to the `imageListFile`, we would also need the `manifests.json`: 

https://github.com/openshift-kni/lifecycle-agent/pull/727/files#diff-8200c824bb53076107616f57fa19be0cb56e51514a600a70c65f0c8a94116c8fR249-R256

Reply:
Or just move ibi_preparation.ibi_preparation to common and use it in both places

Reply:
I have updated the changes to better handle common file paths between IBU/IBI and removed this arg from this function. PTAL